builders/get-started/eth-compare/tx-fees.md

```diff
@@ -257,6 +257,9 @@ contract SayHello {
     Transaction Fee = (GasPrice * TransactionWeight) / {{ networks.moonbase.tx_weight_to_gas_ratio }}
     ```
 
+!!! 请注意
```

Comment:
```suggestion
!!! 注意事项
```

This has to match what is defined in the `moonbeam-mkdocs` repo, so that the css understands and renders the correct color

You can see where it is defined here: https://github.com/papermoonio/moonbeam-mkdocs/blob/master/material-overrides/assets/stylesheets/moonbeam.css#L627

Same for the other admonitions like remember: https://github.com/papermoonio/moonbeam-mkdocs/blob/master/material-overrides/assets/stylesheets/moonbeam.css#L680

and challenge (not used a whole lot): https://github.com/papermoonio/moonbeam-mkdocs/blob/master/material-overrides/assets/stylesheets/moonbeam.css#L663